projects/errors/src/modules/error/error.component.ts

```diff
@@ -19,8 +19,8 @@ import { SkyErrorType } from './error-type';
 export class SkyErrorComponent implements OnInit {
   /**
    * Specifies an error type to use a set of pre-defined values for the image,
-   * title, and description. The valid options are `'broken'`, `'construction'`,
-   * `'notfound'`, and `'security'`.
+   * title, and description. The valid options are `broken`, `construction`,
+   * `notfound`, and `security`.
```

Comment:
Is there a reason you used back ticks here instead of double quotes? `SkyErrorType` resolves to strings - so we should probably use double quotes here too.